tests/mail/tests.py

```diff
@@ -85,6 +85,15 @@ def test_multiple_recipients(self):
         self.assertEqual(message['From'], 'from@example.com')
         self.assertEqual(message['To'], 'to@example.com, other@example.com')
 
+    def test_no_to_recipients(self):
+        email = EmailMessage('Subject', 'Content', 'from@example.com', cc=['cc@example.com'])
+        message = email.message()
+        self.assertEqual(message['Subject'], 'Subject')
```

Comment:
Do you think checking the Subject and Message adds value? It seems to distract from the main issue that's being tested.